Canvases/workflows/copy_canvas.ts

```diff
@@ -0,0 +1,47 @@
+import { DefineWorkflow, Schema } from "deno-slack-sdk/mod.ts";
+
+/**
+ * This workflow demonstrates how to create a canvas and copy it.
+ * Note: Copy canvas only work for enterprise grid workspaces
+ *
+ * To run the workflow, you need to have the following scopes in "botScopes" property in `manifest.ts` file:
+ * - "canvases:read"
+ * - "canvases:write"
+ *
```

Comment:
What do you think about adding a link to the canvas related docs in the comments

```suggestion
 *
 * Learn more about the copy canvas slack function: https://api.slack.com/reference/functions/canvas_copy
```

Reply:
Added on the readme but will be great to also have on the comment. Let me add those